src/forms/FormTextInput.react.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import * as React from "react";
+
+type Props = {|
+  +isPassword?: boolean,

```

Comment:
For a text input field I would replace this with a `+type?: string` and then give it a default value of 'text'. That way the same component could be used for any input type; text, password, number, email etc

Reply:
Good call!  Making this change now.